autosklearn/ensembles/ensemble_selection.py

```diff
@@ -194,8 +194,10 @@ def _bagging(self, predictions, labels, fraction=0.5, n_bags=20):
         return np.array(order_of_each_bag)
 
     def predict(self, predictions):
-        non_null_weights = (weight for  weight in self.weights_ if weight > 0)
-        for i, weight in enumerate(non_null_weights):
+        #non_null_weights = (weight for weight in self.weights_ if weight > 0)
+        #for i, weight in enumerate(non_null_weights):
+        #    predictions[i] *= weight
+        for i, weight in enumerate(self.weights_):
             predictions[i] *= weight
         return np.sum(predictions, axis=0)
```

Comment:
Could this code maybe be simplified a lot more to `return np.average(predictions, axis=0, weights=self.weights_)`?